pkg/utils/release/controller.go

```diff
@@ -109,29 +113,107 @@ func (s *SuiteController) GetPipelineRunInNamespace(namespace, releaseName, rele
 	return nil, err
 }
 
-// GetRelease returns the release with the given name in the given namespace.
-func (s *SuiteController) GetRelease(releaseName, releaseNamespace string) (*v1alpha1.Release, error) {
-	release := &v1alpha1.Release{}
+// CreateReleasePlan creates a new ReleasePlan using the given parameters.
+func (s *SuiteController) CreateReleasePlan(name, devNamespace, application, managedNamespace, AutoReleaseLabel string) (*appstudiov1alpha1.ReleasePlan, error) {
+	var releasePlan *appstudiov1alpha1.ReleasePlan = &appstudiov1alpha1.ReleasePlan{
+		ObjectMeta: metav1.ObjectMeta{
+			Name:      name,
+			Namespace: devNamespace,
+		},
+		Spec: appstudiov1alpha1.ReleasePlanSpec{
+			DisplayName: name,
+			Application: application,
+			Target: kcp.NamespaceReference{
+				Namespace: managedNamespace,
+			},
+		},
+	}
+
+	if AutoReleaseLabel != "" { // AutoReleaseLabel in ReleasePlan is not missing
+		releasePlan.ObjectMeta.Labels[appstudiov1alpha1.AutoReleaseLabel] = AutoReleaseLabel
+	}
+	return releasePlan, s.KubeRest().Create(context.TODO(), releasePlan)
+}
+
+// GetReleasePlan returns the releasePlan with the given name in the given namespace.
+func (s *SuiteController) GetReleasePlan(releasePlanName, devNamespace string) (*appstudiov1alpha1.ReleasePlan, error) {
```

Comment:
I think the argument name should just be `namespace` like I have commented on other functions

Reply:
Updated